src/main/java/com/reshiftsecurity/plugins/intellij/actions/HelpAction.java

```diff
@@ -78,7 +82,19 @@ void actionPerformedImpl(
 							final String info = createProductInfo().toString();
 							CopyPasteManager.getInstance().setContents(new StringSelection(info));
 						} else {
-							BrowserUtil.browse(evt.getURL());
+							URI eventURI = URI.create(evt.getURL().toString());
+							if (eventURI.getScheme().equalsIgnoreCase("mailto")) {
+								Desktop desktop = Desktop.getDesktop();
+								if (desktop.isSupported(Desktop.Action.MAIL)) {
```

Comment:
so what happens if it's not supported? is it a dead action?

Reply:
Copy to clipboard always occurs a few lines above. Compose email is if it is supported.